quinn-proto/src/connection/assembler.rs

```diff
@@ -1,5 +1,5 @@
 use std::{
-    cmp::Ordering,
+    cmp::{max, min, Ordering},
```

Comment:
Are those still required?

Reply:
Yes, in the threshold calculation.

Reply:
I replaced the other uses of `min` and `max` with their method equivalent and got rid of the imports.